tw.ddnet.ddnet.metainfo.xml

```diff
@@ -6,26 +6,36 @@
     <name>DDNet</name>
     <launchable type="desktop-id">tw.ddnet.ddnet.desktop</launchable>
     <summary>
-        DDraceNetwork, a cooperative racing mod of Teeworlds
+        Hardest co-op 2D platformer ever
     </summary>
     <description>
         <p>
-            DDraceNetwork is an actively maintained version of DDRace,
-            a Teeworlds modification with a unique cooperative gameplay.
-            Help each other play through custom maps with up to 64 players,
-            compete against the best in international tournaments, design your
-            own maps, or run your own server.
+            DDraceNetwork (DDNet) is an actively maintained version of DDRace, 
+            a Teeworlds modification with a unique cooperative gameplay. 
+            Help each other play through custom maps with up to 64 players, 
+            compete against the best in international tournaments, design your own maps, 
+            or run your own server. The official servers are around the world. 
+            All ranks made on official servers are available worldwide and you can collect points!
         </p>
     </description>
     <screenshots>
         <screenshot type="default">
-            <image>https://ddnet.org/screenshots/1.png</image>
+            <image>https://raw.githubusercontent.com/flathub/tw.ddnet.ddnet/master/screenshots/1.jpg</image>
```

Comment:
Github doesn't allow hotlinking/being used as a CDN, I believe.

Reply:
A lot of Flathub repositories store screenshots in this way. Flathub only fetches once and then keeps the images on its server. You can inspect the image links here: https://flathub.org/apps/tw.ddnet.ddnet.